Tests/aweXpect.Reflection.Tests/Filters/ThatConstructor.HasParameter.Chaining.Tests.cs

```diff
@@ -0,0 +1,105 @@
+using System.Reflection;
+using Xunit.Sdk;
+
+namespace aweXpect.Reflection.Tests.Filters;
+
+public sealed partial class ThatConstructor
```

Comment:
The tests files should be located in the root directory of the aweXpect.Reflection.Tests project (alongside the existing partial class `ThatMethod`).